Sorting/ArraysOperations.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace SortingAndOutput
+{
+    public static class ArraysOperations
+    {
+        public static void CreatingArray(sbyte[] set)
+        {
+            var random = new Random();
+            for (int i = 0; i < 7; i++)
```

Comment:
nit про порассуждать.

```suggestion
            for (int i = 0; i < {ОДИН_ИЗ_ВАРИАНТОВ_УЛУЧШИТЬ_ТУТ}; i++)
```

{ВТОРОЙ ВАРИАНТ} - если уж метод назван `CreatingArray`, а не `FillArray`, то пускай бы возращал новый массив

в идеале метод должен быть самодостаточным
что если оригинальный массив set содержит 5 элентов. или 14

Reply:
Переименовал метод в **FillArray** и поменял строку на
for (int i = 0; i < set.Length; i++)